### ISSUE TITLE
Suggestions for Improving CityGaussian Output with Sparse Aerial COLMAP Data?

### ISSUE THREAD

**Issue by NiranjanaaR** [Linketic/CityGaussian#145]
Hi team! I'm working on a master's thesis using CityGaussian, and I'm currently testing it on an aerial dataset collected from a drone . The COLMAP reconstruction looks like this (see attached), where the cameras follow two circular downward trajectories and result in a somewhat sparse 3D point cloud.

The problem I'm facing is that in the final rendering from CityGaussian, a large portion of the point cloud seems to be unused or filtered out, and the result includes cloudy artifacts and missing details—especially in the side-view buildings. (Attached another image of the output.)

I suspect the sparsity or orientation of the COLMAP data might be a factor. So my questions are:

Does CityGaussian have known limitations or default filtering behaviors that reduce the quality of sparse datasets?

Is there a way to fine-tune CityGaussian for sparse aerial datasets (e.g., lower point density, tilted cameras)?

Would modifying specific parameters in the config or increasing coverage in COLMAP (even artificially) help?

Any guidance or best practices would be greatly appreciated. Thank you for sharing this awesome work! 

![Image](https://github.com/user-attachments/assets/0792f0b5-ab7d-4608-9f31-ef0b825d6601)

![Image](https://github.com/user-attachments/assets/bff17aa6-c37c-4e12-91ae-06ad0d3eae33)

**Comment by DekuLiuTesla** [comment]
@NiranjanaaR It seems there is already many cameras and no severe content missing is observed. It shows that the overall pipeline seems to works fine. The Cloudy artifacts may because you are trying to extrapolate novel views. Since 3DGS would overfit the training data, if deviation from training view is severe, you may face considerable distortion like this.  If this is the case, try adding more views, or introducting more regularization like depth loss in our V2.
